emails/models.py

```diff
@@ -294,6 +294,12 @@ def has_premium(self):
     def has_phone(self):
         if not self.fxa:
             return False
+        if settings.RELAY_CHANNEL != "prod" and not settings.IN_PYTEST:
+            try:
+                phone_flag = Flag.objects.get(name="phones")
+            except Flag.DoesNotExist:
+                return False
+            return phone_flag.is_active_for_user(self.user)
```

Comment:
On stage and dev, this means that the `free_phones` flag and subscriptions will not be checked. So, if you have the flag turned on, you've also bought phones, and if you don't, you don't have phones.

Reply:
Updated `has_phone` to continue to other checks when the user has the phone flag.